editoast/src/models/fixtures.rs

```diff
@@ -97,27 +114,66 @@ pub fn simple_train_schedule_base() -> TrainSchedule {
         .expect("Unable to parse test train schedule")
 }
 
+pub fn create_created_exception_with_change_groups(key: &str) -> PacedTrainException {
+    PacedTrainException {
+        key: key.into(),
+        exception_type: ExceptionType::Created {},
+        disabled: false,
+        train_name: Some(TrainNameChangeGroup {
+            value: "exception_train_name".into(),
+        }),
+        constraint_distribution: Some(ConstraintDistributionChangeGroup {
+            value: Distribution::Mareco,
+        }),
+        initial_speed: Some(InitialSpeedChangeGroup { value: 10.0 }),
+        labels: Some(LabelsChangeGroup {
+            value: vec!["Label 1".to_string(), "Label 3".to_string()],
+        }),
+        options: Some(OptionsChangeGroup {
+            value: TrainScheduleOptions::default(),
+        }),
+        path_and_schedule: Some(PathAndScheduleChangeGroup {
+            power_restrictions: vec![],
+            schedule: vec![],
+            path: vec![],
+            margins: Margins {
+                boundaries: vec![],
+                values: vec![MarginValue::Percentage(5.0)],
+            },
+        }),
+        rolling_stock: Some(RollingStockChangeGroup {
+            rolling_stock_name: "TJV".into(),
+            comfort: Comfort::AirConditioning,
+        }),
+        rolling_stock_category: Some(RollingStockCategoryChangeGroup { value: None }),
+        speed_limit_tag: Some(SpeedLimitTagChangeGroup {
+            value: Some(NonBlankString("GB".into())),
+        }),
+        start_time: Some(StartTimeChangeGroup {
+            value: DateTime::<Utc>::from_str("2025-05-05T20:05:00+02:00").unwrap(),
+        }),
+    }
+}
+
+pub fn create_modified_exception_with_change_groups(
+    key: &str,
+    occurrence_index: i32,
+) -> PacedTrainException {
+    let mut exception = create_created_exception_with_change_groups(key);
+    exception.exception_type = ExceptionType::Modified { occurrence_index };
+    exception
+}
+
 pub fn simple_paced_train_base() -> PacedTrain {
     let train_schedule_base =
         serde_json::from_str(include_str!("../tests/train_schedules/simple.json"))
             .expect("Unable to parse test train schedule");
     PacedTrain {
         train_schedule_base,
-        exceptions: vec![PacedTrainException {
-            key: "exception_key".into(),
-            exception_type: ExceptionType::Created {},
-            disabled: false,
-            constraint_distribution: None,
-            initial_speed: None,
-            labels: None,
-            options: None,
-            path_and_schedule: None,
-            rolling_stock: None,
-            rolling_stock_category: None,
-            speed_limit_tag: None,
-            start_time: None,
-            train_name: None,
-        }],
+        exceptions: vec![
+            create_created_exception_with_change_groups("exception_key_1"),
+            create_modified_exception_with_change_groups("exception_key_2", 0),
+        ],
```

Comment:
let's keep the "simple" paced train exception free

Reply:
It's not important so I don't mind resolving and merging as is, but please write a comment on why you disagree.